### ISSUE TITLE
Why the output_type of preprocessor cannot be the tf.float64?

### ISSUE THREAD

**Issue by Davidsirui** [keras-team/autokeras#761]
### Bug Description
When I set the output_type of Normalization and ImageAugmentation to tf.float32, the whole pipeline runs well. But it fails when I set it to tf.float64.

### Reproducing Steps
  * Step 1: ...
  * Step 2: ...

### Setup Details
Include the details about the versions of:
 - OS type and version:
 - Python: 
 - autokeras: 
 - scikit-learn:
 - numpy:
 - keras:
 - scipy:
 - tensorflow:
 - pytorch:
